From 82861909c79ad262597a0fc05847bf97f22feb03 Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Thu, 28 Sep 2017 16:43:13 +0200
Subject: [PATCH 06/15] Set drbd_device in block device.

For now this should do the trick .. however in the end we want to
have 2 block devices, one for DRBD and one for lower backing devs.
---
 drbd/drbd_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drbd/drbd_main.c b/drbd/drbd_main.c
index 63526f5..4e11aa2 100644
--- a/drbd/drbd_main.c
+++ b/drbd/drbd_main.c
@@ -3515,6 +3515,7 @@ enum drbd_ret_code drbd_create_device(struct drbd_config_context *adm_ctx, unsig
 	device->this_bdev = bdget(MKDEV(DRBD_MAJOR, minor));
 	/* we have no partitions. we contain only ourselves. */
 	device->this_bdev->bd_contains = device->this_bdev;
+	device->this_bdev->drbd_device = device;
 
 	init_bdev_info(q->backing_dev_info, drbd_congested, device);
 
-- 
2.8.0-rc4

